packages/web/components/home/hooks.ts

```diff
@@ -0,0 +1,114 @@
+import { TokenHistoricalPrice } from "@osmosis-labs/stores/build/queries-external/token-historical-chart/types";
```

Comment:
Let's use the type from the server file instead. Be sure to use `import type` to only import the TS type.